data_platform/ingestion/sync_reddit.py

```
@@ -591,46 +588,30 @@ def process_task(task: RedditTask, entry: dict[str, Any]) -> None:
-    comment_storage = RedditStorageManager(StorageStage.RAW, dataset_id)
-    post_storage = comment_storage.post_storage()
-
-    ingestion_params = config["ingestion_params"]
-    sync_tasks = build_sync_tasks(ingestion_params)
+def _validate_reddit_config(config: dict[str, Any]) -> None:
     record_types: list[str] = config["record_types"]
     include_comments = COMMENTS_RECORD_TYPE in record_types
     include_posts = POSTS_RECORD_TYPE in record_types
-
     if not include_comments and not include_posts:
         raise ValueError(f"Unsupported record types for checkpoint sync: {record_types}")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)